webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc

```diff
@@ -0,0 +1,33 @@
+== *Create, Share and Consume Devfiles*
+:imagesdir: ../../../images
+'''''
+The following is an example of a devfile ecosystem that is framed within the context of an organization looking to standardize their development environment.  In the simplest case, developers can just consume the devfiles that are available from the public community registry.  If the organization sees the need to author and share their own devfiles internally, they will need a role based approach where Developers, Devfile Authors, and Registry Administrators can interact together.
+
+image::devfileEcosystem.svg[]
+
+
+=== *The Main Areas of the Devfile Ecosystem*
+=== *Create*
+
+A devfile author aka “stack provider” can be an individual or a group representing a runtime vendor.  This role requires sound knowledge of the supported runtime so devfiles can be created with the proper guidance on how to build and run applications.
```

Comment:
`aka` is too informal. Simply say, `A devfile author or "stack provider"...` or `A devfile author, also known as "stack provider," can be...` 

For the second sentence, I'd rephrase so that the focus is more on the role. IBM SG wants us to focus on the user whenever we can. What do you think of this rewrite?: 

`Devfile authors need sound knowledge of the supported runtime so they can create devfiles to build and run applications.`

Reply:
I decided to use the term `runtime provider` so we're consistent with the persona title.